nuxeo-runtime/nuxeo-launcher-commons/src/test/resources/configurator/nuxeo.conf

```diff
@@ -21,3 +21,12 @@ nuxeo.wizard.done=false
 test.sampled.prop2=someValue
 
 nuxeo.force.generation=true
+
+nuxeo.db.host=${env:NUXEO_DB_HOST}
+
+org.nuxeo.fake.vindoz=${env?:NUXEO_UNKNOWN_VAR}
```

Comment:
I still have an issue with the syntax: in most of other languages `?:param` does not mean `Boolean.parseBoolean(param)`. See https://en.wikipedia.org/wiki/Elvis_operator

If you need that feature, what about the `??` ? 
See https://en.wikipedia.org/wiki/Null_coalescing_operator and http://freemarker.org/docs/dgui_template_exp.html#dgui_template_exp_missing_test
FreeMarker could be considered as a preferred reference since it's used in the templates.
For your example, that would result in: `nuxeo.redis.enabled=${env??NUXEO_REDIS_HOST}`